ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/source-item/source-item.tsx

```diff
@@ -116,11 +117,16 @@ export default hot(module)(({ id, sourceActions, available }: Props) => {
                       sourceAction.description
                     }`}
                     onClick={() => {
-                      lightboxInstance.model.updateTitle(sourceAction.title)
-                      lightboxInstance.model.open()
-                      lightboxInstance.showContent(
-                        new SourceAppView({ url: sourceAction.url })
-                      )
+                      if (sourceAction.id.startsWith('catalog.data.source.window')) {
```

Comment:
❓ Could we key off of a new field instead of relying on the naming convention of the id?

Reply:
We rely on this pattern of looking at action id prefixes to determine certain things in the ui elsewhere, specifically for list actions.

Reply:
https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/transformer/impl/EmbeddedListMetacardsHandler.java#L108

Reply:
That's true, but it still seems like a misuse of the id